relayer/src/chain.rs

```diff
@@ -1,49 +1,46 @@
-pub(crate) mod cosmos;
-pub use cosmos::CosmosSDKChain;
-
-pub mod handle;
-pub mod runtime;
-
-#[cfg(test)]
-pub mod mock;
-
-use crossbeam_channel as channel;
 use std::{sync::Arc, thread};
-use tokio::runtime::Runtime as TokioRuntime;
 
+use crossbeam_channel as channel;
 use prost_types::Any;
-
-use tendermint_proto::Protobuf;
-
 // TODO - tendermint deps should not be here
 use tendermint::account::Id as AccountId;
 use tendermint::block::Height;
+use tendermint_proto::Protobuf;
+use tokio::runtime::Runtime as TokioRuntime;
```

Comment:
FYI: There may be conflicts here if we merge https://github.com/informalsystems/ibc-rs/pull/565 before this PR.